### PR TITLE
Weekly events report nudge

### PR DESCRIPTION
**Pull request by abdullai-t**
####  Summary / Highlights
Link [ticket](https://github.com/massenergize/api/issues/586)
Currently the nudge for weekly report is scheduled for 8am every Monday.
#### Testing Steps (Provide details on how your changes can be tested)

- Run `full-celery-local` to start Django server, celery instance and celery-beat
- Start the admin portal server and add some preferences on the settings page if you have not done so
- toggle some from live to offline and live again
- Navigate to  `_main_/celery/app` and alter the `schedule` property of `send_weekly_event_nudge` to a short period(next 10mins)
- Wait for the period to elapse and check your email.
- 

#### Requirements (place an `x` in each `[ ]`)
* [x] I've read and understood the [Contributing Guidelines](/blob/main/.github/contributing.md) and have done my best effort to follow them.
* [x] I've tested my changes manually.
* [ ] I've added unit tests to my PR.

##### Transparency ([Project board](https://github.com/orgs/massenergize/projects/7/views/2))
* [x] I've given my PR a meaningful title.
* [x] I linked this PR to the relevant issue.
* [x] I moved the linked issue from the "Sprint backlog" to "In progress" when I started this.
* [x] I moved the linked issue to "QA Verification" now that it is ready to merge.

### CODE REVIEW

**Comment by Davidjwkinghorn** [comment]
I'd like to merge the two migration files here if that wouldn't break anything. `event.updated_at` was added then removed, so I removed it and replaced it with `event.live_at` in the first migration file. The effect should be the same and there should be no issue with any non-reversible migrations. Let me know,  though, @abdullai-t if you think it could cause any issues and we can roll it back.

I'd change migration 0123 from
"""
from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('database', '0122_auto_20221012_1857'),
    ]

    operations = [
        migrations.AddField(
            model_name='event',
            name='created_at',
            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='event',
            name='updated_at',
            field=models.DateTimeField(auto_now=True),
        ),
    ]
"""
to
"""
from django.db import migrations, models
import django.utils.timezone


class Migration(migrations.Migration):

    dependencies = [
        ('database', '0122_auto_20221012_1857'),
    ]

    operations = [
        migrations.AddField(
            model_name='event',
            name='created_at',
            field=models.DateTimeField(auto_now_add=True, default=django.utils.timezone.now),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='event',
            name='live_at',
            field=models.DateTimeField(blank=True, null=True),
        ),
    ]
"""
Migration 0124 is deleted.

**Comment by abdullai-t** [comment]
@Davidjwkinghorn  i dont think it will cause any issues

**Comment by Davidjwkinghorn** [comment]
Would it be possible to make the events display on cards to look like this?
Event Email Nudge

Events should be displayed in this email if they are scheduled to show up in the next two months (or 60 days if that works better). This would be instead of showing events by when they go live.

**Comment by abdullai-t** [comment]
@Davidjwkinghorn  - post mark template update
Screenshot 2022-11-22 at 9 17 25 PM

**Comment by Davidjwkinghorn** [comment]
Let's do some more styling on that email before we make it live. Other than that, though, I think it's looking good. What's the best way to test this? Is there a way to generate and trigger the email manually?

**Comment by Davidjwkinghorn** [comment]
@kaatvds, @apowelka, @BradHN1 do you have thoughts on the styling for the email? Should we try to make it look like the admin portal with the purple and blue or something else?

**Comment by kaatvds** [comment]
Re styling, I think just put the MassE logo up top. Keep it simple.

**Comment by abdullai-t** [comment]
@kaatvds  and @Davidjwkinghorn - I have updated the postmark template.
```
{
	"name": "Abdullai Tahiru",
	"events": [
		{
			"logo": "https://www.massenergize.org/wp-content/uploads/2021/08/energize-wayland.jpg",
			"view_link": "https://www.massenergize.org/communities/",
			"title": "Heating and Cooling with Electric Heat Pumps",
			"location": "In Person",
			"date": "1st Dec, 2022",
			"community": "Wayland",
			"share_link": "https://www.massenergize.org/communities/"
		},
		{
			"logo": "https://massenergize-prod-files.s3.amazonaws.com/media/Framingham2020_smaller.jpg",
			"view_link": "https://www.massenergize.org/communities/",
			"title": "Heating and Cooling with Electric Heat Pumps",
			"location": "In Person",
			"date": "1st Dec, 2022",
			"community": "Framingham",
			"share_link": "https://www.massenergize.org/communities/"
		},
		{
			"logo": "https://www.massenergize.org/wp-content/uploads/2021/08/energize-lexington.jpg",
			"view_link": "https://www.massenergize.org/communities/",
			"title": "Heating and Cooling with Electric Heat Pumps",
			"location": "Online",
			"date": "1st Dec, 2022",
			"community": "Lexington",
			"share_link": "https://www.massenergize.org/communities/"
		}
	],
	"change_preference_link": "https://www.massenergize.org/communities/"
}

```

**Comment by Davidjwkinghorn** [comment]
Nice, this is coming together!

**Comment by abdullai-t** [comment]
Done. But will have to pull down @BradHN1 preference PR code in.

**Comment by abdullai-t** [comment]
@Davidjwkinghorn , @BradHN1 , @frimpongopoku - PR is ready for review

**Review by BradHN1 — APPROVED** [review]
Good work

File: src/_main_/celery/app.py [hunk 1]
```diff
@@ -14,6 +14,10 @@
 
 
 app.conf.beat_schedule = {
+    'send_weekly_event_nudge': {
```
[the remark this answers is not in the document]
Comment (abdullai-t) [reply src/_main_/celery/app.py]: I removed it

[the remark this answers is not in the document]
Comment (abdullai-t) [reply src/_main_/celery/app.py]: The daily task will be scheduled from the admin portal

**Pull request merged**